tests/phpunit/TemplateTest.php

```diff
@@ -24,8 +24,8 @@ protected function tearDown() {
   
   protected function process_citation($text) {
     $template = new Template();
```

Comment:
Do we need a `$page= new Page()`?

Reply:
that’s an excellent question.   The new Page() is made in process_page()
Here is the code with subroutine kind of inlined 

    $template = new Template();
    $page = $this->process_page($text);
         $page = new Page();
         $page->parse_text($text);
         $page->expand_text();
    $template->parse_text($page->parsed_text());
    return $template;